smsdk/client_v0.py

```diff
@@ -118,10 +116,12 @@ def convert_to_valid_url(
     valid_url = f"{protocol}://{domain}"
 
     if port:
-        valid_url = f"{valid_url}:{port}"
+        # valid_url = f"{valid_url}:{port}"
+        log.warning(f"Ignored the user specified port.")
```

Comment:
I think we should support port specification.  If a developer is using port forwardi or some other mechanism to use an unexpected port, they are supported. Adding support for this should not be too difficult. It should just be a wiring issue.

Reply:
Done.